v2/stacks/common.mk

```diff
@@ -37,7 +37,7 @@ verify-service: $(LOCAL_ENV_FILE)
 	exit 4
 
 .PHONY: up
-up: verify-service
+up: init verify-service
```

Comment:
Is this not a breaking change for other services in the stack that use verify-service?

Reply:
I don't think so. When we run the `make up` command it will execute both of those targets and not only init, isn't  it ?

Reply:
What about other stacks?  I would create a new target for `init verify-service` just incase